packages/office-ui-fabric-react/src/components/pickers/Suggestions/Suggestions.tsx

```diff
@@ -321,8 +321,12 @@ export class Suggestions<T> extends BaseComponent<ISuggestionsProps<T>, ISuggest
       }
     );
 
+    const selectedIndex = suggestions.findIndex(suggestion => suggestion.selected);
```

Comment:
Unfortunately, `findIndex` is not supported by IE and so it can't be used just yet.

Something like 
```
    let selectedIndex = -1;
    suggestions.some((element, index) => {
      if (element.selected) {
        selectedIndex = index;
        return true;
      }
      return false;
    });
```

Should be able to work as a substitute!

Reply:
@joschect why the `selectedIndex` is even incremented when we have a `resultsMaximumNumber` prop set. Shouldn't the Search for more link be next accessible by the keyboard?

Reply:
I would assume if `resultsMaximumNumber` prop is set then we need to make sure somewhere higher in the code hierarchy that on clicking the down keyboard arrow will not increment the index of the selected suggestion. Instead, the user would have to either tab to reach the searchForMore link and activate it. @joschect am I missing some important design implementation and the author is actually submitting the correct fix in here?

Reply:
That would solve our issue as well (if the focus went straight to 'Search more matches' after scrolling down).

Reply:
@Vitalius1 I think you're correct. I'll take another look. I think that this behavior is appropriate if there is no "Search more matches" button but if there is then you're correct that it should get focused.

Reply:
@joschect @Vitalius1 I'll investigate if I can make that change

 (if scrolling past resultsMaximumNumber, focus should shift to 'searchForMore', otherwise continue scrolling)

Reply:
@joschect what is SearchForMore doing when clicked? Is it replacing the results with a new set of suggestions? Or it just adds them to the existing ones? How is `resultsMaximumNumber` playing a role in that case?

Reply:
It triggers a callback which makes it's way out to the consuming component. It's up the consuming component how those results are returned, it might return all the current suggestions plus a few more, or it could return a completely different set.

It's related to the resultsMaximumNumber because one of it's uses is to allow the user to search for different results if not all of them are present.

Reply:
I have some other sprint stuff at the moment. But next week I can modify this PR so that instead of scrolling down the list of suggestions, the focus will shift to 'searchForMore' component (if present).

Reply:
I've done a bit of digging and I'm pretty sure there's a bug somewhere in the tryHandleKeyDown function. I believe that it needs to check to see if it's reached the maximum of items and there's a searchForMore button then it should change the action to SuggestionActionType.searchMore. There's some other stuff missing and it looks like it can get focus if the parent were to implement a few things. However we should probably try to handle it automatically somehow.

Reply:
I think for now this PR is probably a step in the right direction and likely we should get this merged, and then investigate focusing the searchForMore button properly. I suspect it will be a bit larger of a change. @dzearing  or @cliffkoh  do you have any opinions about leaving something like that for a later pr?